web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -2167,4 +2168,51 @@ describe("<BusinessFormationPaginator />", () => {
       page.fillText("Search business name", "Pizza Joint");
     };
   });
+
+  describe("remembers formation step", () => {
+    it("shows the first formation step with default formationStep", () => {
+      preparePage(initialUserData, displayContent);
+      expect(screen.getByText(Config.formation.fields.businessName.header)).toBeInTheDocument();
+    });
+
+    it("shows the last formation step when lastVisitedPage has step index 4", () => {
+      const legalStructureId = "limited-liability-company";
+      const profileData = generateFormationProfileData({ legalStructureId });
+      const formationData = generateFormationData({ lastVisitedPageIndex: 4 });
+      displayContent = {
+        formationDbaContent: generateFormationDbaContent({}),
+      };
+      initialUserData = generateUserData({ profileData, formationData });
+      preparePage(initialUserData, displayContent);
+      expect(screen.getByText(Config.formation.general.submitButtonText)).toBeInTheDocument(); // expect submit button
+    });
+
+    it("formState.lastVistedPage updates on step change", async () => {
+      const page = preparePage(initialUserData, displayContent);
+      expect(screen.getByText(Config.formation.fields.businessName.header)).toBeInTheDocument();
+
+      await page.stepperClickToBillingStep();
+      await waitFor(() => {
+        expect(currentUserData().formationData.lastVisitedPageIndex).toEqual(3);
+      });
+
+      await page.stepperClickToBusinessStep();
+      await waitFor(() => {
+        expect(currentUserData().formationData.lastVisitedPageIndex).toEqual(1);
+      });
+    });
+
+    it("defaults to first formation step we go beyond page index allowed", () => {
```

Comment:
it looks like we're protecting for edge case when greater than but not less than. can we add one for -1?